app/src/main/java/com/esri/arcgismaps/kotlin/sampleviewer/model/Sample.kt

```diff
@@ -109,7 +109,7 @@ data class Sample(
             // Otherwise, function should be modified to return list of URLs for each image
             val modifiedJsonSampleName = sampleName.replace(" ", "-").lowercase()
             val imageFileName = imageArray.first().toString().replace("\"", "")
-            return "https://raw.githubusercontent.com/Esri/arcgis-maps-sdk-kotlin-samples/v.next/$modifiedJsonSampleName/$imageFileName"
+            return "https://raw.githubusercontent.com/Esri/arcgis-maps-sdk-kotlin-samples/v.next/samples/$modifiedJsonSampleName/$imageFileName"
```

Comment:
was this an intentional change?

Reply:
@sorenoid Yes, this change allows us to render screenshots in the sample viewer by grabbing the image from the `v.next` branch which now lives in `samples/<samples-name>/<file-name>`

> Fixed a minor issue with screenshots not loading due to v.next branch change.